samples/workspace-codelab.js

```diff
@@ -0,0 +1,172 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// [START_EXCLUDE]
+// Instructions for this codelab can be found on this page
+// https://cloud.google.com/channel/docs/codelabs/workspace/provisioning
+// [END_EXCLUDE]
+
+// [START channel_credentials]
+const {JWT} = require('google-auth-library');
+const {grpc} = require('google-gax');
+const {CloudChannelServiceClient} = require('@google-cloud/channel');
```

Comment:
Maybe you can add more comments here, for example, that you're instantiating a client here, etc.